examples/client.rs

```diff
@@ -118,9 +118,9 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         eprintln!("Response: {:?} {}", resp.version(), resp.status());
         eprintln!("Headers: {:#?}", resp.headers());
 
-        while let Some(chunk) = stream.recv_data().await? {
+        while let Some(mut chunk) = stream.recv_data().await? {
             let mut out = tokio::io::stdout();
-            out.write_all(&chunk).await.expect("write_all");
+            out.write_buf(&mut chunk).await.expect("write_all");
```

Comment:
is this meant to stay write_all, just like here https://github.com/hyperium/h3/pull/67#discussion_r768213749 ?

Reply:
It will end up being "write_all_buf()"

Reply:
Is this change supposed to be included? It looks like we want it to stay `write_all_buf`.

Reply:
I actually messed up the rebase. Good catch, thanks!